src/FSharp.Stats/Vector.fs

```diff
@@ -120,6 +128,7 @@ module Vector =
     ///Returns length of vector
     let length (vector:vector) = VG.length vector
     ///Returns length of vector
+    [<Obsolete("Do not use. Use [length] instead.")>]
     let nrows (vector:vector) = VG.length vector
```

Comment:
Since in mathematics the number of rows of a vector is reasonable, this second naming of the Vector.length function is more intuitive in some cases.